### ISSUE TITLE
Update mpg321 Library for Python 3 Compatibility

### ISSUE THREAD

**Issue by stefets** [4br3mm0rd/mpyg321#46]
The current version of the mpg321 library relies on deprecated Python 2.7 and Python 3.x syntax. With Python 2.7 deprecated, it's essential to update mpg321 to ensure compatibility with Python 3 for ongoing support and security.

Proposed Changes:
- Update the codebase to replace deprecated Python 2.7 syntax with Python 3 equivalents.
- Modify any incompatible libraries or dependencies to support Python 3.

Expected Benefits:
- Ensure mpg321 remains usable and accessible in Python 3 environments.
- Future-proof the library against Python 2.7 deprecation and end-of-life issues.

Steps to Implement:
- Identify and replace deprecated syntax and libraries.

Thanks!

**Comment by 4br3mm0rd** [comment]
I believe that the library is completely working with python2 and python3. I personally run it under python3 and I don't face any issues.
In the beginning I didn't want to add some more python3 features in order to support both versions (because I believe we can afford that). But if you really want it we can add some typing and python3 specific code

**Comment by stefets** [comment]
Hi! there is at least a Python 3 specific syntax in the lib.
* Enum added in V3.4 https://docs.python.org/3/library/enum.html found in consts.py

It's not a priority. And I wouldn't be surprised if there is a code analyzer that can check the syntax and suggest corrections. I can take care of it in the next few weeks.

**Comment by 4br3mm0rd** [comment]
You're right, I didn't think of it when I implemented that. OK so let's move to full python3 then :-)

**Comment by stefets** [comment]
Ok! I will do the changes sometime after the changes we are working on.

**Comment by JamesRandom** [comment]
It looks like this can be closed :)

**Comment by stefets** [comment]
Make sense since the actual codebase is incompatible with Python 2.*
